README.md

```diff
@@ -118,7 +132,7 @@ entities (no need to draw an ERD)...
 Your answer...
 
 ```
-Replace this with your answer
+It is a relational database used to connect large collections of information
```

Comment:
An ERD is how we describe the models in our domain. Our database is structured based off of this but is not the ERD itself